deployment/k8sPaiLibrary/maintainlib/update.py

```diff
@@ -245,9 +249,33 @@ def run(self):
         self.node_list_from_k8s = self.get_node_list_from_k8s_api()
         self.node_config_from_cluster_conf = self.get_node_config_from_cluster_configuration()
         self.node_config_from_k8s = self.get_node_config_from_k8s()
+        if self.node_config_from_k8s == None:
+            self.logger.warning("No node configuration is found in your kubernetes cluster.")
+            self.logger.warning("Paictl will replace it with the node configuration from cluster-configuration.")
+            self.logger.warning("Suggestion: Due to the first you execute this command.")
+            self.logger.warning("Suggestion: This time please don't change (Remove or add) the machine list in your configuration.")
```

Comment:
User may be confused, it's may need to clarify the difference between`node configuration` and `cluster-configuration`.

Reply:
User will never know there is a machine-list.